src/main/java/com/venafi/vcert/sdk/policy/converter/cloud/CloudPolicyToPolicyConverter.java

```diff
@@ -0,0 +1,186 @@
+
+        List<String> subjectCNRegexes = cit.sanDnsNameRegexes();
+        if (subjectCNRegexes != null && subjectCNRegexes.size() > 0 && !subjectCNRegexes.get(0).equals(".*"))
+            getSubjectAltNamesFromPolicySpecification(policySpecification).dnsAllowed(true);
+    }
+
+    private void processDefaults( PolicySpecification policySpecification, CloudPolicy cloudPolicy ) throws Exception {
+        if ( cloudPolicy.certificateIssuingTemplate().recommendedSettings() != null ) {
+
+            CertificateIssuingTemplate.RecommendedSettings recommendedSettings = cloudPolicy.certificateIssuingTemplate().recommendedSettings();
+
+            processDefaultsSubject( policySpecification, recommendedSettings);
+
+            processDefaultsKeyPair( policySpecification, recommendedSettings);
+        }
+    }
+
+    private void processDefaultsSubject( PolicySpecification policySpecification, CertificateIssuingTemplate.RecommendedSettings recommendedSettings ) throws Exception {
+        if ( recommendedSettings.subjectOValue() != null )
+            getDefaultsSubjectFromPolicySpecification( policySpecification ).org( recommendedSettings.subjectOValue() );
+
+        if ( recommendedSettings.subjectOUValue() != null ) {
+            String[] subjectOUValues = {recommendedSettings.subjectOUValue()};
+            getDefaultsSubjectFromPolicySpecification(policySpecification).orgUnits(subjectOUValues);
+        }
+
+        if ( recommendedSettings.subjectLValue() != null )
+            getDefaultsSubjectFromPolicySpecification( policySpecification ).locality( recommendedSettings.subjectLValue() );
+
+        if ( recommendedSettings.subjectSTValue() != null )
+            getDefaultsSubjectFromPolicySpecification( policySpecification ).state( recommendedSettings.subjectSTValue() );
+
+        if ( recommendedSettings.subjectCValue() != null )
+            getDefaultsSubjectFromPolicySpecification( policySpecification ).country( recommendedSettings.subjectCValue() );
+    }
+
+    private void processDefaultsKeyPair( PolicySpecification policySpecification, CertificateIssuingTemplate.RecommendedSettings recommendedSettings) throws Exception {
+        processDefaultsKeyType( policySpecification, recommendedSettings);
+    }
+
+    private void processDefaultsKeyType( PolicySpecification policySpecification, CertificateIssuingTemplate.RecommendedSettings recommendedSettings) throws Exception {
+
+        if( recommendedSettings.key() != null ) {
+            if( recommendedSettings.key().type() != null )
+                getDefaultsKeyPairFromPolicySpecification(policySpecification).keyType(recommendedSettings.key().type());
+
+            if( recommendedSettings.key().length() != null )
+                getDefaultsKeyPairFromPolicySpecification(policySpecification).rsaKeySize(recommendedSettings.key().length());
+        }
+    }
+
+}
```

Comment:
No new line at end of file

Reply:
In fact that doesn't affect the functionallity of the class